### DISCUSSION TITLE
layouts folder missing content

### DISCUSSION THREAD

**Discussion by Tokosan** [thuliteio/doks#1251] [Q&A]
I installed the theme using the

`npm create hyas@latest -- --template doks`

command, and the layouts folder only contains

```
layouts
│   index.html
└───partials
    ├───footer
    │       script-footer-custom.html
    └───head
            custom-head.html
            script-header.html
```

which doesn't allow me to change, for example, the guide layout. I've seen other discussions talking about how to modify some things, but they all rely on things that doesn't exist anymore, like the [Add to docs menu](https://getdoks.org/tutorial/add-menus/#add-to-docs-menu) page and a lot of other things.

Am I missing something?

How can I get the old (Doc 0.5) layout folder so I can edit things?

**Comment by h-enk** [comment]
Copy the file you'd like to override from the `@hyas/doks-core`  module and paste in your project's root. 

For example, if you'd like to customize the homepage, copy `node_modules/@hyas/doks-core/layouts/index.html` and paste as `/layouts/index.html`. Make your changes in the latter.
